### ISSUE TITLE
Chrome extension

### ISSUE THREAD

**Issue by jeroen7s** [plaintextoffenders/plaintextoffenders#6]
I created a basic chrome extension using the data from this repository to display a warning on sites that don't store passwords securely
can be found here: https://github.com/jeroen7s/plaintext-offenders-chrome
feedback is appreciated, as this is the first time i'm developing a chrome extension

**Comment by jeroen7s** [comment]
now also supports firefox

**Comment by RokeJulianLockhart** [comment]
@jeroenev, IDK whether [`issues/16`](https://github.com/plaintextoffenders/plaintextoffenders/issues/16#issue-617426355) continues to affect uptake of this.

**Comment by jeroenev** [comment]
Yeah I'm aware of that, but I doubt they'll update it since the site is effectively dead.
Also not sure if the extension still works as I haven't used it in years 🙃 
Though it loads the list from Github directly so should still work

**Comment by RokeJulianLockhart** [comment]
@jeroenev, if the website's source was publicly available, I'd submit a PR. However, although I'm in communication with some of the project owners to gain access to some of this project, accessing the website source appears unexpectedly difficult. I'll update this when, depending upon how [`issues/5#issuecomment-3767597660`](https://github.com/plaintextoffenders/plaintextoffenders/issues/5#issuecomment-3767597660) progresses.

**Comment by jeroenev** [comment]
> accessing the website source appears unexpectedly difficult

Isn't the website built on top of Tumblr? I guess that's why you can't get the source-code
at least every post on the site has this:

> Short URL for this post: http://tumblr.com/whatever

ex: http://plaintextoffenders.com/post/15614153845/peopleanswers-com-people-answers-employee

**Comment by RokeJulianLockhart** [comment]
@jeroenev, Urgh; I'm unfamiliar with Tumblr, and most of those other (in retrospect, WYSIYG) website hosts. I thought it was like WordPress. Thanks.
